configs/config.yaml

```diff
@@ -9,6 +9,6 @@ server:
 data:
   spiceDb:
     useTLS: false
-    endpoint: "${ENDPOINT:spicedb:50051}"
+    endpoint: "localhost:50051"
```

Comment:
Keep it spicedb, as the docker setup uses the same config file.

Reply:
Yep, that was unintended. Should be reverted now.